appcenter-push/android/src/main/java/com/microsoft/appcenter/reactnative/push/AppCenterReactNativePushModule.java

```diff
@@ -35,7 +35,7 @@ public AppCenterReactNativePushModule(Application application) {
         AppCenterReactNativeShared.configureAppCenter(application);
         if (AppCenter.isConfigured()) {
             boolean startPush = true;
-            boolean enablePushInJavascript = AppCenterReactNativeShared.getConfiguration().optBoolean(ENABLE_PUSH_IN_JAVASCRIPT);
+            boolean enablePushInJavascript = AppCenterReactNativeShared.readConfigurationFile(application).optBoolean(ENABLE_PUSH_IN_JAVASCRIPT);
```

Comment:
Why changing that? It should be read only once. Plus it will crash if returning null here, that's why we were using new object. We already log the error.

Reply:
https://github.com/microsoft/appcenter-sdk-react-native/pull/711/commits/97b82b2801f5ad8a59e776b683380faed9e98c1f now returns empty JSONObjection when appcenter-config.json can't be parsed. This avoids checking null of `readConfigurationFile` in this file.

Reply:
`readConfigurationFile` can be called multiple times but only parsing the file once, see https://github.com/microsoft/appcenter-sdk-react-native/pull/711/files#diff-69dc9df643336331d14c703104f712f5R88.

In fact, AppCenterReactNativeShared's `readConfigurationFile` API and `configureAppCenter` API can be called multiple times in any order but only paring config file once.